contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -515,6 +515,22 @@ var EditMetadataView = BaseViews.BaseEditorView.extend({
 			/* TODO: FIX THIS LATER TO APPEND TAG VIEWS TO AREA*/
 			this.$el.find("#tag_area").append("<div class='col-xs-4 tag'>" + this.$el.find("#tag_box").val().trim() + " <span class='glyphicon glyphicon-remove pull-right delete_tag' aria-hidden='true'></span></div>");
 			this.$el.find("#tag_box").val("");
+			//tags should be an array of tag names
+			tags = [];
+			tags[0] = 'hi';
+			tags[1] = 'jordan';
+			Backbone.ajax({
```

Comment:
@jayoshih  you can place this Backbone.ajax call anywhere you need to create the ContentTag objects. If there usage more than once, might be good to wrap this into a function.